Assets/Scripts/BottomMenuManager.cs

```
@@ -0,0 +1,84 @@
+using UnityEngine;
+
+public class BottomMenuManager : MonoBehaviour
+{
+    public BottomMenu[] bottomMenus;
+
+    void Start()
+    {
+        if (bottomMenus == null || bottomMenus.Length == 0)
+        {
+            Debug.LogWarning("No bottom menus assigned!");
```

Why log a warning? You could throw an exception or just use a serialized field and it'll automatically log an exception if nothing has been set